interface/reports/sales_by_item_new.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * Sales by Item Report - Modernized Controller
+ *
+ * This is a report of sales by item description.
+ * Modernized version using service layer and Twig templates.
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @author    Terry Hill <terry@lillysystems.com>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Stephen Waite <stephen.waite@cmsvt.com>
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2006-2016 Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2015-2016 Terry Hill <terry@lillysystems.com>
+ * @copyright Copyright (c) 2017-2018 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2025      Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2025      Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once("$srcdir/patient.inc.php");
+require_once "$srcdir/options.inc.php";
```

Comment:
Why parentheses on line 25, but none on line 26?

To make the phpstan warning go away, ask warp to add an appropriate `/** @var */` declaration, since srcdir is defined in ../globals.php.